### ISSUE TITLE
vertical metrics check summary

### ISSUE THREAD

**Issue by RosaWagner** [fonttools/fontspector#208]
- [ ] fonttools/fontspector#206 
- [x] fonttools/fontbakery#4139
- [x] **WARN** sum of typo metrics value is greater than UPM `hhea/typoAscender + (-hhea/typoDescender) ≥ UPM` / rationale can recommend 120 to 130% of UPM size (fixed in fonttools/fontbakery#3762)
- [ ] fonttools/fontspector#207 
- [x] fonttools/fontbakery#3241
- [ ] fonttools/fontbakery#3819 Particularly needed for complex scripts (non Latin). See [#3326 comment](https://github.com/googlefonts/fontbakery/issues/3326#issuecomment-1243057039)

**Comment by felipesanches** [comment]
I wonder if we should actually split this into multiple new checks, or if there's a benefit in checking all those things at once in a single check.

I suspect that having individual checks could be better because it would allow us to have more detailed rationales for each without resulting in a ridiculously long amount of text (like having the entire vertical metrics part of the gf-spec in a single rationale). 

Checking more than a single thing in a given check would only be helpful if such things were tightly related to each other and could be explained in a single rationale without resulting in something excessively verbose.

**Comment by m4rc1e** [comment]
> Caps are centered : capsHeight - hhea/typoAscender ~= - hhea/typoDescender (this is only recommended for fonts with Latin, Greek or Cyrillic as "main" script) (WARN)

What is "capsHeight"? I'm assuming it is https://docs.microsoft.com/en-us/typography/opentype/spec/os2#scapheight.

**Comment by RosaWagner** [comment]
Yes it is, I am used to Glyphs naming stuff. (If the value correspond to the actual capital height of the actual outline)

**Comment by m4rc1e** [comment]
> I wonder if we should actually split this into multiple new checks, or if there's a benefit in checking all those things at once in a single check.

@felipesanches I'll start with a single check and split it into individual checks if it hurts too much.

**Comment by felipesanches** [comment]
thanks, @m4rc1e

**Comment by felipesanches** [comment]
hey, @m4rc1e, This issue is currently assigned to the upcoming release (v0.8.9). Would you like to postpone it to a future release milestone or will you plan to implement this in the short term so that it can be included in v0.8.9? The upcoming release will probably happen at some point later this month, ideally mid-April.

**Comment by felipesanches** [comment]
The fontspector issue tracker does not currently use priority/importance labels, but @felipesanches had labeled this as "P0 urgent" on May 25, 2021, when it was hosted at the fontbakery issue tracker.
